tests/integration/fixtures.py

```diff
@@ -1,23 +1,35 @@
 import logging
+import os
 from typing import TYPE_CHECKING
 
+import boto3
+import botocore.config
 import pytest
 
 from localstack.utils.aws import aws_stack
 from localstack.utils.aws.aws_stack import create_dynamodb_table
 from localstack.utils.common import short_uid
 
 if TYPE_CHECKING:
+    from mypy_boto3_cloudformation import CloudFormationClient
     from mypy_boto3_dynamodb import DynamoDBClient
     from mypy_boto3_s3 import S3Client
     from mypy_boto3_sns import SNSClient
     from mypy_boto3_sqs import SQSClient
+    from mypy_boto3_ssm import SSMClient
 
 LOG = logging.getLogger(__name__)
 
 
 def _client(service):
-    return aws_stack.connect_to_service(service)
+    if os.environ.get("TEST_TARGET") == "AWS_CLOUD":
```

Comment:
Can we encapsulate this check (`os.environ.get("TEST_TARGET") == "AWS_CLOUD"`) into a small config/util function? (and also use it in `integration/conftest.py`)

(I'd generally try to avoid accessing `os.environ` directly across the codebase. I'm aware that we're already doing this in a few places, but maybe we can start pulling things out - this will also help us create more systematic config/environment abstractions as we go...)

Reply:
@whummer  My thinking here was that the ones used in our tests often aren't supposed to be part of the user-facing localstack config. In this case it's mostly just for debugging and checking our integration tests for parity. Adding this to the user config would just add unnecessary bloat to it IMO. 
If we find a use case for it in the core though, I'd agree that it should be part of the core config.

Reply:
Fair point. Better not add it to the core user config - we can pick this up as part of a general config overhaul (user + dev configs) in the near future... 👍